charts/promitor-agent-scraper/templates/deployment.yaml

```diff
@@ -31,6 +31,13 @@ spec:
         checksum/secret: {{ include (print $.Template.BasePath "/secret.yaml") . | sha256sum }}
         {{- end }}
     spec:
+      {{- if .Values.rbac.create }}
+      {{- if .Values.rbac.serviceAccount.create }}
+      serviceAccountName: {{ template "promitor-agent-scraper.fullname" . }}
```

Comment:
Same here, we have to check if `.Values.rbac.serviceAccount.name` is specified and use that, if not use `{{ template "promitor-agent-scraper.fullname" . }}`; no?